content/en/docs/concepts/scheduling-eviction/scheduling-framework.md

```diff
@@ -84,8 +84,10 @@ the Pod is put into the active queue or the backoff queue
 so that the scheduler will retry the scheduling of the Pod.
 
 {{< note >}}
-QueueingHint evaluation during scheduling is a beta-level feature and is enabled by default in 1.28. 
-You can disable it via the
+QueueingHint evaluation during scheduling is a beta-level feature. 
+It's enabled by default in 1.28.0 - 1.28.5, 
+but is disabled by default in 1.29 because of excessive memory footprint.
```

Comment:
```suggestion
The v1.28 release series initially enabled the associated feature gate; however, after the
discovery of an excessive memory footprint, the Kubernetes project set that feature gate
to be disabled by default. In Kubernetes {{< skew currentVersion >}}, this feature gate is
disabled and you need to enable it manually.
```
Let's try this. Once the v1.28.6 release is out, we'll want to edit this message and backport something.